jvm/sargon-android/build.gradle.kts

```diff
@@ -79,6 +79,7 @@ dependencies {
     implementation("net.java.dev.jna:jna:5.13.0@aar")
     implementation("androidx.annotation:annotation:1.7.1")
     implementation("androidx.compose.ui:ui-tooling-preview-android:1.6.2")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Comment:
@micbakos-rdx this fixed a Gradle 9 depreciation warning I had which prevented pre commit hook from passing

All tests pass now, so I hope this is correct.